test/Macros/lit.local.cfg

```diff
@@ -2,7 +2,7 @@
 if 'asserts' not in config.available_features:
     config.unsupported = True
 
-config.subsitutions = list(config.substitutions)
+config.substitutions = list(config.substitutions)
```

Comment:
This might cause a change in behavior. Are we certain it's correct?

Reply:
Looks very much like a typo in a line that did not prove necessary.

@compnerd Do we need this line removed or corrected?